environments/staging/common/rds.tf

```diff
@@ -0,0 +1,52 @@
+module "postgres" {
+  source = "../../common/rds"
+
+  environment    = var.environment
+  name           = "TradeTariffPostgres${title(var.environment)}"
+  engine         = "postgres"
+  engine_version = "13.11"
+
+  deletion_protection = false # while configuring
+
+  # smallest that supports encryption at rest and postgres 13.11
+  instance_type      = "db.t3.micro"
+  backup_window      = "22:00-23:00"
+  maintenance_window = "Fri:23:00-Sat:01:00"
+  private_subnet_ids = data.terraform_remote_state.base.outputs.private_subnet_ids
+
+  allocated_storage     = 20
```

Comment:
Its worth checking the current db storage and occupied space in `cf` and align to that